src/facilities-context/FacilitiesContext.tsx

```diff
@@ -89,6 +97,31 @@ export const FacilitiesProvider: React.FC<{ children: React.ReactNode }> = ({
     }
   }, [scenarioId, state.facilities, state.rtData]);
 
+  // fetch shadow facilities based on user facilities
+  const shouldFetchShadow = useShadowDataEligible();
+  useEffect(() => {
+    // clean up any existing shadow facility data
+    facilitiesActions.clearShadowFacilities(dispatch);
+
+    if (!shouldFetchShadow) return;
```

Comment:
Do we still want to perform the `clearShadowFacilities` call above if this is false?

Reply:
we do ... this value is unique per scenario, so it could flip from true to false when, e.g., you change from your baseline scenario to one of your other ones. I can see that it's not particularly intuitive though, I was actually struggling a bit with the best way to do this ... I think I might factor out the cleanup to a different effect that is closer to its cause